test/config.sh

```diff
@@ -174,11 +174,13 @@ globalExcludeTests+=(
 	# single-binary images
 	[hello-world_utc]=1
 	[nats_utc]=1
+        [nats-streaming_utc]=1
 	[swarm_utc]=1
 	[traefik_utc]=1
 
 	[hello-world_no-hard-coded-passwords]=1
 	[nats_no-hard-coded-passwords]=1
+        [nats-streaming_no-hard-coded-passwords]=1
```

Comment:
Ok done. I looked for all references to `nats` in the file and I think that these are the only 3 ;-)
Hope it is good now. Thanks again for your patience!